src/qibosoq/rfsoc_server.py

```diff
@@ -42,7 +37,7 @@ def execute_program(data: dict, qick_soc: QickSoc) -> dict:
         (dict): dictionary with two keys (i, q) to lists of values
     """
     opcode = OperationCode(data["operation_code"])
-    args = ()
+    args = (_ for _ in ())  # initialize empty generator for sequences
```

Comment:
type(()) is tuple, but for sweepers is a generator and mypy complains. In this way is an empty generator

Reply:
Can you point me where the sweepers' counterpart is defined?

Reply:
Here it is:
https://github.com/qiboteam/qibosoq/blob/d0bd2e2c63862df4e0736a6ca3542cf73ecd9f74/src/qibosoq/rfsoc_server.py#L49

Maybe I can use a tuple here, to avoid the weirdness of the empty generator

Reply:
Having added a function to convert sweepers from dict to sweeper objects with arrays, this is just a a list so it should be fine simply with
```python
args = []
```